releasenotes/notes/0.24/preset-pm-vf2-max-trials-958bb8a36fff472f.yaml

```
@@ -0,0 +1,25 @@
+---
+upgrade_transpiler:
+  - |
+    The maximum number of trials evaluated when searching for the best
+    layout using :class:`.VF2Layout` and :class:`.VF2PostLayout` is now
+    limited in
+    :func:`~qiskit.transpiler.preset_passmanagers.level_1_pass_manager`,
+    :func:`~qiskit.transpiler.preset_passmanagers.level_2_pass_manager`,
+    and
+    :func:`~qiskit.transpiler.preset_passmanagers.level_3_pass_manager`
+    to ``2,500``, ``25,000``, and ``250,000``, respectively. Previously,
```

To my eye the comma makes this look like 2.5 but maybe that's just me -- feel free to ignore 🙂 
```suggestion
    to ``2 500``, ``25 000``, and ``250 000``, respectively. Previously,
```